### PR TITLE
Use new tracking client for logging metrics

### PR DESCRIPTION
**Pull request by kesamet**

### CODE REVIEW

File: multiclass/bedrock.hcl [hunk 1]
```diff
@@ -2,48 +2,44 @@
...
-    install = [
-        "pip install --upgrade pip",
-        "pip install -r requirements.txt",
-    ]
-    script = [
-        {sh = [
-            "/app/entrypoint.sh"
-        ]}
-    ]
+  image = "basisai/express-flask:v0.0.3"
```
Comment (sweatybridge) [inline multiclass/bedrock.hcl:27]:
```suggestion
  image = "basisai/express-flask:v0.0.7"
```

**Review by sweatybridge — APPROVED** [review]
LGTM. I will push a new version of the express image too.

File: binary_class/doc/requirements.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
 tensorflow
-bdrk[analyzer,model-monitoring]>=0.9,<0.10
+bdrk[analyzer]>=0.9.1,<0.10
```
Comment (sweatybridge) [inline binary_class/doc/requirements.txt:2]:
```suggestion
bdrk[analyzer]>=0.9.1,<0.10
boxkite==0.0.5
```

do we need boxkite here?

File: regression/bedrock.hcl [hunk 3]
```diff
@@ -2,47 +2,43 @@
...
-            "pip install --upgrade pip",
-            "pip install -r requirements.txt",
-        ]
-        script = [{sh = ["python train.py"]}]
-        resources {
-            cpu = "4"
-            memory = "4G"
-        }
+  step "train_regression" {
+    image = "python:3.9"
```
Comment (sweatybridge) [inline regression/bedrock.hcl:6]:
```suggestion
    image = "python:3.7"
```

are we using python 3.7 everywhere?

File: binary_class/doc/requirements.txt (hunk 2 above)
Comment (kesamet) [reply binary_class/doc/requirements.txt:2]: not using model-monitoring, so no need for boxkite

File: regression/bedrock.hcl (hunk 3 above)
Comment (kesamet) [reply regression/bedrock.hcl:6]: no, it's up to the author of the example

**Pull request merged**